training/distributed_training/pytorch/model_parallel/gpt-j/11_train_gptj_smp_tensor_parallel_notebook.ipynb

```diff
@@ -0,0 +1,988 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    # training_dataset_location = YOUR_S3_BUCKET/training
</pre><p>Style: please use brackets around any variables that the user would edit, ie <code>&lt;YOUR_S3_BUCKET&gt;</code> (2+ occurrences) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>